src/settings-ui/Settings.UI/SettingsXAML/Views/EnvironmentVariablesPage.xaml

```diff
@@ -6,7 +6,7 @@
     xmlns:controls1="using:CommunityToolkit.WinUI.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:ui="using:CommunityToolkit.WinUI.UI"
```

Comment:
Version 7.1.2 uses MDL2

Reply:
Great catch, yeah we shouldn't reference these any more.